### PR TITLE
feat!: remove goerli

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Remove Goerli from supported networks

# Testing

- Unit tests
- Publish a local version and test it on cowswap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ const orderSigningUtils = new OrderSigningUtils()
 ### Sign, fetch, post and cancel order
 
 For clarity, let's look at the use of the API with a practical example:
-Exchanging `0.4 GNO` to `WETH` on `Goerli` network.
+Exchanging `0.4 GNO` to `WETH` on `Gnosis chain` network.
```
Comment (alfetopito) [inline README.md:47]: Migrated examples to Gnosis chain for future proofing.
Sepolia will eventually die and we'll have to do this all over again.
It was the same with Rinkeby before Goerli.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/cow-sdk",
-  "version": "5.0.0",
+  "version": "6.0.0",
```
Comment (alfetopito) [inline package.json:3]: Major version bump!

File: src/common/chains.ts [hunk 3]
```diff
@@ -4,7 +4,6 @@
  */
 export enum SupportedChainId {
   MAINNET = 1,
-  GOERLI = 5,
```
Comment (alfetopito) [inline src/common/chains.ts:7]: That's the main change

**Review by anxolin — APPROVED** [review]
Looks good to me

**Pull request merged**